tmuxinator.gemspec

```diff
@@ -40,7 +40,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.4.6"
 
   s.add_dependency "erubis", "~> 2.6"
-  s.add_dependency "thor", "~> 0.19", ">= 0.15.0", "< 1.0.0"
+  s.add_dependency "thor", "~> 1.0"
```

Comment:
what changed that this is the right thing to do?  perhaps we should document whatever it is in the changelog?

Reply:
The initial "fix" didn't actually address the underlying "problem". Thor releasing v1+ breaks tmuxinator on Arch because the package's build description file doesn't specify any version constraints for the Arch ruby-thor package -- it uses Arch packages in place of tmuxinator's runtime dependencies as managed via RubyGems.

In this case, the actual fix which accommodates the Arch package is pretty minimal and tmuxinator was already compatible with Thor's new API, but it is debatable whether this sort of issue should be addressed in tmuxinator or the Arch package.

Reply:
Any additional thoughts, @adamstrickland? The Arch folks seem happy with this and it should be a seamless upgrade for people using RubyGems.